llama_hub/file/xml/README.md

```diff
@@ -0,0 +1,19 @@
+# XML Loader
+
+This loader extracts the text from a local XML file. A single local file is passed in each time you call `load_data`.
+
+## Usage
+
+To use this loader, you need to pass in a `Path` to a local file.
+
+```python
+from pathlib import Path
+from llama_index import download_loader
+
+XMLReader = download_loader("XMLReader")
+
+loader = XMLReader()
+documents = loader.load_data(file=Path('../example.xml'))
+```
+
+This loader is designed to be used as a way to load data into [LlamaIndex](https://github.com/run-llama/llama-hub/tree/main/llama_hub) and/or subsequently used as a Tool in a [LangChain](https://github.com/hwchase17/langchain) Agent. See [here](https://github.com/emptycrown/llama-hub/tree/main) for examples.
```

Comment:
I think you updated the wrong link. You updated the Llamaindex link. 

The link that was needed to be updated is the third one. My bad 😸